cmd/podman/pod_inspect.go

```diff
@@ -39,9 +39,6 @@ func podInspectCmd(c *cliconfig.PodInspectValues) error {
 	var (
 		pod *libpod.Pod
 	)
-	if err := checkMutuallyExclusiveFlags(&c.PodmanCommand); err != nil {
```

Comment:
Entirely gone?

Reply:
it doesnt have an --all so it doesnt need that verification